src/routes/auth.js

```diff
@@ -90,13 +90,14 @@ router.get(
         if (err) {
```

Comment:
it was necessary to make them as dummy test because we are deleting some product and orders which was making them to fail but the end points are working perfectly